src/freenet/node/Node.java

```diff
@@ -2539,6 +2533,46 @@ public void realRun() {
 		System.out.println("Node constructor completed");
 	}
 
+	// peers-offers/*.fref files
+	private void processPeersOffersFrefFiles(SubConfig nodeConfig, int configOptionSortOrder) {
```

Comment:
This method both handles the configuration and processes the files but its name does not reflect that. Either change the method or change the name. :)